server/rpc_test.go

```diff
@@ -559,14 +559,14 @@ func StubBroadcastSessionsManager() *BroadcastSessionsManager {
 	}
 }
 
-func TestSelect(t *testing.T) {
+func TestSelectSession(t *testing.T) {
```

Comment:
We already have a `TestSelectOrchestrator` which has issues with the match done if running `go test -run Select`. This was mostly problematic because SelectOrchestrator is currently not concurrent safe under `-race`.